city_scrapers/spiders/alle_county.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+
+import urllib3
+from datetime import datetime, timedelta
+from legistar.events import LegistarEventsScraper
+from city_scrapers.constants import BOARD, FORUM
+from city_scrapers.spider import Spider
+
+
+class AlleCountySpider(Spider):
+    name = 'alle_county'
+    agency_name = 'Allegheny County Government'
+    timezone = 'America/New_York'
+    allowed_domains = ['alleghenycounty.legistar.com']
+    START_URL = 'https://alleghenycounty.legistar.com'
+    start_urls = [START_URL]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows
+        the `Open Civic Data event standard
+        <http://docs.opencivicdata.org/en/latest/data/event.html>`.
+
+        Change the `_parse_id`, `_parse_name`, etc methods
+        to fit your scraping needs.
+        """
+        events = self._make_legistar_call()
+        return self._parse_events(events)
+
+    def _make_legistar_call(self, since=None):
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        les = LegistarEventsScraper(requests_per_minute=0)
+        les.EVENTSPAGE = self.START_URL + '/Calendar.aspx'
+        les.BASE_URL = self.START_URL
+        if not since:
+            since = datetime.today().year
+        return les.events(since=since)
+
+    def _parse_events(self, events):
+        for item, _ in events:
+            start_time = self._parse_start(item)
+            end_time = self._parse_end(item)
+            data = {
+                'timezone': self.timezone,
+                '_type': 'event',
+                'name': self._parse_name(item),
+                'event_description': '',
+                'all_day': self._parse_all_day(item),
+                'classification': self._parse_classification(item),
+                # _generate_id takes `start` instead of start_time.
+                # `start` is only used internally.
+                'start': start_time,
+                'start_time': start_time,
+                'end_time': end_time,
+                'end': end_time,
+                'location': self._parse_location(item),
+                'documents': self._parse_documents(item),
+                'sources': self._parse_sources(item),
+            }
+            data['id'] = self._generate_id(data)
+            data['status'] = self._generate_status(data, '')
```

Comment:
From looking at Legistar and [another Legistar-based scraper](https://github.com/City-Bureau/city-scrapers/blob/master/city_scrapers/spiders/cook_board.py#L55), canceled meetings are noted in the meeting location, so that should be passed to `_generate_status` here